src/app/pages/profile/profile.module.ts

```diff
@@ -9,13 +9,20 @@ import { PersonalInformationComponent } from 'src/app/pages/profile/personal-inf
 import { ProfileSidebarComponent } from 'src/app/pages/profile/profile-sidebar/profile-sidebar.component';
 import { ProfileComponent } from 'src/app/pages/profile/profile.component';
 import { RoleTransformPipe } from 'src/app/shared/pipes/role.pipe';
+import { TransactionInformationComponent } from './transaction-information/transaction-information.component';
+
+import { ComponentsModule } from 'src/app/components/components.module';
+import { EmptyTransactionComponent } from './transaction-information/empty-transaction/empty-transaction.component';
```

Comment:
Hey @Jaman-dedy, we should use absolute imports

Reply:
Thanks @CryceTruly , I worked on that